content/working_with/console/widgets/blueprintCatalog.md

```diff
@@ -7,13 +7,13 @@ draft: false
 
 Displays blueprints from a repository under a configurable Github account or from an HTTP endpoint providing JSON response.
 
-By default, the widget presents the blueprints under specific . It is configured to read the blueprints URLs from a JSON file. 
+By default, the widget presents the blueprints listed in JSON file taken from widget configuration. 
  
 The widget includes an upload option for each of the blueprints, which lets you easily upload it to the current tenant on the manager. 
 After uploading a blueprint from the catalog, you will be able to see it under the **Blueprints** widget.
 
 You can also point the widget to read from a GitHub user account. 
-You can then filter the presented blueprints by providing a filter query in the widget’s settings.  
+You can then filter the presented blueprints by providing a filter query in the widget’s settings. Check [Seraching for repositories](https://help.github.com/en/github/searching-for-information-on-github/searching-for-repositories) page to get more information on how to filter data in GitHub.
```

Comment:
Typo: Should be "searching", not "seraching"

Reply:
Fixed.